paasta_tools/cli/cmds/validate.py

```diff
@@ -417,6 +421,58 @@ def validate_autoscaling_configs(service_path):
     return returncode
 
 
+def validate_secrets(service_path):
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    system_paasta_config = load_system_paasta_config()
+    return_value = True
+    for cluster in list_clusters(service, soa_dir):
+        vault_env = system_paasta_config.get_vault_cluster_config().get(cluster)
```

Comment:
no need to call `system_paasta_config.get_vault_cluster_config()` in every iteration

Reply:
you're right, will move this call outside the for loop.